src/Conda.jl

```diff
@@ -70,14 +70,18 @@ function python_dir(env::Environment)
 end
 const PYTHONDIR = python_dir(ROOTENV)
 
-# note: the same conda program is used for all environments
-const conda = if Sys.iswindows()
-    p = script_dir(ROOTENV)
-    conda_bat = joinpath(p, "conda.bat")
-    isfile(conda_bat) ? conda_bat : joinpath(p, "conda.exe")
-else
-    joinpath(bin_dir(ROOTENV), "conda")
+if ! @isdefined(CONDA_EXE)
+    # We have an oudated deps.jl file that does not define CONDA_EXE
```

Comment:
This is impossible. If there's an outdated deps.jl that means there was a update of Conda.jl which means the package has to be built first.

Reply:
It's possible in a few situations such as using multiple Julia installations with the same `JULIA_DEPOT_PATH` and/or a modified `JULIA_LOAD_PATH`. Julia will only force a rebuild if it is aware that the version changed.

For example, I did this to myself in the course of testing the package by using `pkg"dev Conda"`. In this case, the Conda package was checked out in `~/.julia/dev/Conda` and the deps file at `~/.julia/dev/Conda/deps/deps.jl`was carried over between several versions as I moved between branches or I changed environments.

While we could just throw and error if we detect this condition, encouraging the user to rebuild, just doing the former calculation seems friendlier since we can just follow the old behavior here.

We see this issue with an outdated deps.jl file quite commonly with GR.jl and users will appear in support channels complaining that plotting is broken. The issues were reduced once we just started fixing the issues automatically.

Reply:
We can remove it now, but my recommendation is to keep it for the moment and then remove it at a later time to smooth over the migration process.